Kentico.Kontent.Management/IManagementClient.cs

```diff
@@ -32,19 +34,19 @@ public interface IManagementClient
         /// Cancels publishing of the language variant.
         /// </summary>
         /// <param name="identifier">The identifier of the language variant identifier of which publishing should be canceled.</param>
-        Task CancelPublishingOfLanguageVariant(LanguageVariantIdentifier identifier);
+        Task CancelPublishingOfLanguageVariantAsync(LanguageVariantIdentifier identifier);
```

Comment:
This is a breaking change, it might be worth discussing suffix Async...

Reply:
Agreed - might be better to do this in separate PR - review would be much easier

Reply:
Definitely agree that separate PR would be nicer.